flamingpy/examples/performances.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
```

Comment:
Let's make a local backup of this file and remove it from this PR

Reply:
The reason is, that while these benchmarks are interesting, we don't think they benefit public users. Let's keep them locally and we may publish them late in a different form.